tests/tck/features/steps/Steps.py

```diff
@@ -25,70 +27,97 @@ def step_impl(context):
 def step_impl(context):
     graphs.any_graph()
 
+@then(u'parameters are')
+def set_params(context):
+    global params
+
+    params = {}
+    for row in context.table.rows:
+        params[row[0]]=row[1]
+    print(params)
+
+@given(u'parameters are')
+def set_params(context):
+    global params
+
+    params = {}
+    for row in context.table.rows:
+        params[row[0]]=row[1]
+
```

Comment:
If possible:

```
@then(u'parameters are')
@given(u'parameters are')
def set_params(context):
    global params
    params = {}
    for row in context.table.rows:
        params[row[0]]=row[1]
```

It is strange to see the exact same function, I would expect a redefinition of a function error.

Reply:
anotated one function with both annotations